components/AllAboutSpecialty.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <section class="section bg-navy flex flex-wrap justify-center">
+    <div class="section-content">
+      <h2 class="text-white text-4xl" data-aos="fade-up">
+        {{ text.h2 }}
+      </h2>
+      <section
+        class="
+          grid
+          space-y-12
+          pb-28
+          lg:pt-4 lg:grid-cols-2 lg:gap-32 lg:space-y-0
+        "
+        :class="flipParagraphs ? 'xl:grid-cols-8' : null"
+        data-aos="fade-up"
+        data-aos-delay="200"
+      >
+        <div
+          :class="
+            flipParagraphs
+              ? 'xl:grid xl:col-span-5 order-last pt-12 xl:pt-0'
+              : 'grid'
+          "
+        >
+          <slot name="unboxed-text">
+            <p
+              class="
+                text-white
+                whitespace-pre-line
+                lg:content-center lg:place-self-center
+              "
+            >
+              {{ text.unboxedText }}
+            </p>
+          </slot>
+        </div>
+        <div
+          class="
+            bg-blue-light
+            whitespace-pre-line
+            flex
+            items-center
+            p-12
+            pt-4
+            rounded-md
+            text-navy
+          "
+          :class="flipParagraphs ? 'xl:col-span-3' : null"
+        >
+          <slot name="boxed-text">
+            <p class="mb-0">
+              {{ text.boxedText }}
+            </p>
+          </slot>
```

Comment:
```suggestion
            <p class="mb-0">
                    <slot name="boxed-text"/>
            </p>
```

The same would apply here